install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2341,6 +2341,12 @@ clustermesh:
       # For EKS LoadBalancer, use annotation service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
       annotations: {}
 
+      # -- The externalTrafficPolicy of service used for apiserver access.
+      externalTrafficPolicy: Cluster
+
+      # -- The externalTrafficPolicy of service used for apiserver access.
+      internalTrafficPolicy: Cluster
```

The two comments are identical, I guess this one should say `internalTrafficPolicy`?

```suggestion
      # -- The internalTrafficPolicy of service used for apiserver access.
      internalTrafficPolicy: Cluster
```

---

thx 🙏

---

@qmonnet 
Thx!
Fixed and squashed to one commit.
How do I add your contribution to the commit?